cmd/jb-server/main.go

```diff
@@ -57,6 +57,24 @@ func main() {
 			Usage:  "path to compute cluster that VMs will be booted in",
 			EnvVar: "JUPITER_BRAIN_VSPHERE_CLUSTER_PATH",
 		},
+		cli.IntFlag{
+			Name:   "vsphere-concurrent-read-operations",
+			Usage:  "number of concurrent read vSphere operations",
```

Comment:
I'd like to see us _somehow_ document at this is used by the `Fetch` and `List` functions, but I'm not sure of the right _Go-ish_ way to do this. My first thought is to include that in the `Usage`.